packages/uniforms-antd/src/BaseForm.js

```diff
@@ -1,9 +1,22 @@
-import BaseForm from 'uniforms/BaseForm';
+import BaseForm     from 'uniforms/BaseForm';
+import {PropTypes}  from 'react';
 
 const AntD = parent => class extends parent {
     static AntD = AntD;
 
+    static propTypes = {
+            ...parent.propTypes,
+            fieldData: PropTypes.object
+    };
+
     static displayName = `AntD${parent.displayName}`;
+
+    getChildContextState () {
+        return {
+            ...super.getChildContextState(),
+            fieldData: this.props.fieldData
+        };
```

Comment:
I guess it's here because of #137. I'd rather move it to the `BaseForm`, but not in this PR. And We have to think out a better name.